src/deadline/keyshot_adaptor/KeyShotClient/keyshot_handler.py

```diff
@@ -52,6 +56,10 @@ def start_render(self, data: dict) -> None:
         Raises:
             RuntimeError: .
         """
+
+        selected_render_device = self.render_kwargs["render_device"]
+        print(f"Selected render device: {selected_render_device}")
```

Comment:
Nit: It might be slightly better to print out the render engine since it has both the device and the product vs interior engine info in it. It'd also be nice to print out the engine from lux.getRenderEngine() instead of from the job parameters so we can be more confident it was applied correctly.